components/host-dashboard/reports-section/HostFeesSection.js

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/client';
+import dynamic from 'next/dynamic';
+import { FormattedMessage, useIntl } from 'react-intl';
+const Chart = dynamic(() => import('react-apexcharts'), { ssr: false });
+
+import { get, groupBy } from 'lodash';
+import styled from 'styled-components';
+
+import { formatCurrency } from '../../../lib/currency-utils';
+import { API_V2_CONTEXT, gqlV2 } from '../../../lib/graphql/helpers';
+import { i18nTransactionSettlementStatus } from '../../../lib/i18n/transaction';
+
+import ContainerOverlay from '../../ContainerOverlay';
+import { Box, Flex } from '../../Grid';
+import Loading from '../../Loading';
+import { StyledSelectFilter } from '../../StyledSelectFilter';
+import StyledSpinner from '../../StyledSpinner';
+import { P } from '../../Text';
+
+const mainReportsQuery = gqlV2/* GraphQL */ `
+  query ReportsPageQuery($hostSlug: String!, $startDate: DateTime!, $endDate: DateTime!) {
+    host(slug: $hostSlug) {
+      id
+      createdAt
+      currency
+      hostMetricsTimeSeries(startDate: $startDate, endDate: $endDate, timeUnit: MONTH) {
+        hostFees {
+          nodes {
+            date
+            amount {
+              value
+              valueInCents
+              currency
+            }
+          }
+        }
+        hostFeeShare {
+          nodes {
+            date
+            settlementStatus
+            amount {
+              value
+              valueInCents
+              currency
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+const ChartWrapper = styled.div`
+  position: relative;
+
+  .apexcharts-legend-series {
+    background: white;
+    padding: 8px;
+    border-radius: 10px;
+    & > span {
+      vertical-align: middle;
+    }
+  }
+
+  .apexcharts-legend-marker {
+    margin-right: 8px;
+  }
+`;
+
+const getChartOptions = (intl, hostCurrency) => ({
+  chart: {
+    id: 'chart-host-report-fees',
+    stacked: true,
+  },
+  legend: {
+    show: true,
+    showForSingleSeries: true,
+    horizontalAlign: 'left',
+    fontWeight: 'bold',
+    fontSize: '12px',
+    markers: {
+      width: 16,
+      height: 16,
+    },
+  },
+  dataLabels: { enabled: false },
+  grid: {
+    raw: { opacity: 0 },
+    column: { opacity: 0 },
+    xaxis: { lines: { show: true } },
+    yaxis: { lines: { show: false } },
+  },
+  plotOptions: {
+    bar: {
+      columnWidth: '50%',
+    },
+  },
+  colors: ['#46347F', '#95DDF4', '#F5C451', '#0EA755'], // TODO(HostReport): Use host primary colors
+  xaxis: {
+    categories: [...new Array(12)].map(
+      (_, idx) => `${intl.formatDate(new Date(0, idx), { month: 'short' }).toUpperCase()}`,
+    ),
+  },
+  yaxis: {
+    labels: {
+      minWidth: 40,
+      formatter: function (value) {
+        if (value >= 1000) {
+          return `${Math.round(value / 1000)}k`;
+        } else {
+          return formatCurrency(value, hostCurrency);
+        }
```

Comment:
@Betree : Here my thinking is there's a possibility that some values will have the dollar sign (those which use `formatCurrency`) and other not. Maybe we should remove the dollar sign altogether to be in [sync with design](https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=13722%3A162076).

Reply:
Good catch, I initially did this to make sure we have the currency in the tooltip but we should not display it on the `y` legend. I updated that in https://github.com/opencollective/opencollective-frontend/pull/6849/commits/3cc01d07305311a1bb3f48a030bab9f1fd0660b4

![image](https://user-images.githubusercontent.com/1556356/133404039-992e9d1f-ec9a-4ecb-897c-a136fbd38940.png)